actors/builtin/multisig/multisig_actor.go

```diff
@@ -141,6 +141,10 @@ func (a Actor) Propose(rt vmr.Runtime, params *ProposeParams) *ProposeReturn {
 	rt.ValidateImmediateCallerType(builtin.CallerTypesSignable...)
 	callerAddr := rt.Message().Caller()
 
+	if params.Value.Sign() <= 0 {
```

Comment:
This appears to be checked much later, but it's easier to validate this method if we check it up-front.

Reply:
Zero must be allowed. A message send does not necessarily transfer value, it may be a method call.

Reply:
Ah, this is a proposed invocation of a method, got it.